docs/_static/debug.js

```diff
@@ -1,7 +1,7 @@
-// Add debug actions to flyout menu
-
-$(function () {
-  $("[data-toggle='rst-debug-badge']").on("click", function () {
+// Add debug actions to flyout menu.
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
Shouldn't we create a `domReady()` helper function as we are doing in the community repository?

Reply:
seems simpler to just use the event listener? If you mean to try to load jquery here as well, there is no need, since jquery isn't used till you press the "toggle" button, at that point jquery is already included/loaded.

Reply:
I'm just saying that instead of repeating over and over the `document.addEventListener("DOMContentLoaded", function () {` part we could create a helper function and reuse on the whole file.